src/main/java/cubicchunks/util/FastCubeBlockAccess.java

```diff
@@ -257,4 +249,38 @@ public ClientProxy() {
             return ((CubeProviderClient) prov).getLoadedChunks();
         }
     }
+
+    @Override
+    public void markEdgeNeedLightUpdate(BlockPos pos, EnumSkyBlock type) {
+        if(type == EnumSkyBlock.BLOCK)
+            return;
+        int x = pos.getX();
+        int y = pos.getY();
+        int z = pos.getZ();
+        Cube cube = this.getCube(x, y, z);
+        if (cube != null) {
```

Comment:
invert that if() to `if (cube == null) return;`. Early return IS more readable in many cases (you don't need to scroll all the way down to the end of the if() to see that there is no else statement for example)